bridges/otelzap/core.go

```diff
@@ -123,14 +125,48 @@ func (o *Core) Sync() error {
 	return nil
 }
 
-// TODO
 // Check determines whether the supplied Entry should be logged using core.Enabled method.
```

Comment:
I see this comment comes from zap, but as-is, it looks incorrect, since the method doesn't just check, it also adds itself to the checked entry.

```suggestion
// Check determines whether the supplied Entry should be logged using core.Enabled method.
// If the entry should be logged, the Core adds itself to the CheckedEntry and returns the result.
```